Gemfile

```
@@ -13,6 +13,7 @@ gem 'fabrication' # Fabrication generates objects in Ruby. Fabricators are schem
 gem 'sidekiq' # background processing for Ruby
 gem 'bootsnap', require: false # Reduces boot times through caching; required in config/boot.rb
 gem 'i18n-js', '3.5.1' # A library to provide the I18n translations on the Javascript
+gem 'httparty' #Makes http fun again! Ain't no party like a httparty, because a httparty don't stop.
```

Can we make the comment a bit simple and more introductory?
```suggestion
gem 'httparty' # A HTTP client for Ruby.
```